database/migrations/2020_03_20_105348_alter_resources_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AlterResourcesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('resources', function (Blueprint $table) {
+            $table->dropUnique('resources_title_unique');
+          });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('resources', function (Blueprint $table) {
+            //
```

Comment:
The most minor of minor / nitpicky comments - it's "good practice" to include the opposite effect in the down() method, just so you can rollback migrations and re-do them without issue. 

In reality, you will rarely encounter a problem if you don't do this! But it's a good exercise in learning about the more obscure migration methods...